worldedit-core/src/main/java/com/sk89q/worldedit/command/FlattenedClipboardTransform.java

```diff
@@ -112,10 +113,12 @@ public Region getTransformedRegion() {
      * @return the operation
      */
     public Operation copyTo(Extent target) {
+        //FAWE start
         Extent extent = original;
         if (transform != null && !transform.isIdentity()) {
             extent = new BlockTransformExtent(original, transform);
         }
+        //FAWE end
         ForwardExtentCopy copy = new ForwardExtentCopy(extent, original.getRegion(), original.getOrigin(), target, original.getOrigin());
         copy.setTransform(transform);
         if (original.hasBiomes()) {
```

Comment:
commenting at the end cos this seems like a bad place for this file? fawe -> extent-> transform?

Reply:
Uh, this is an upstream class. We only modified this method.

Reply:
Well upstream have it in a dumb place then